test/plug/router_test.exs

```diff
@@ -178,6 +181,12 @@ defmodule Plug.RouterTest do
     assert conn.resp_body == "root"
   end
 
+  test "dispatch empty body plug" do
+    assert_raise RuntimeError, fn ->
```

Comment:
Yeah, I thought about that, but it seemed a bit awkward to eventually break a test based on an error message.

It would maybe make more sense to have Plug throw an ```InvalidPlugError``` and test for it rather than a specific message?